chapters/arrays.tex

```diff
@@ -470,6 +470,7 @@ \subsection{Reduction Functions and Operators}\label{reduction-functions-and-ope
 \end{lstlisting}\end{synopsis}
 \begin{semantics}
 Returns the least element of the scalars $x$ and $y$; as defined by \lstinline!<!.
+Vectorizable according to \cref{scalar-functions-applied-to-array-arguments}.
```

Comment:
I don't see that we write this out for other functions, so I think the change in `{numeric-functions-and-conversion-functions}` should be sufficient (and better, to not introduce the expectation that we write this out for each individual operator).

```suggestion
```